### PR TITLE
Node 16

### PR DESCRIPTION
**Pull request by timoxley**
Removes `node-webcrypto-ossl` from the client.
Tests all packages on node 16 and 17 instead of just node 16 in CI.

### CODE REVIEW

**Review by juslesan — APPROVED** [review]

File: packages/client/webpack.config.js [hunk 1]
```diff
@@ -113,7 +113,6 @@ module.exports = (env, argv) => {
                 crypto: require.resolve('crypto-browserify'),
                 buffer: require.resolve('buffer/'),
                 'node-fetch': path.resolve('./src/shim/node-fetch.ts'),
```
Comment (juslesan) [inline packages/client/webpack.config.js:115]: Are the 'apostrophies' required here?

Comment (timoxley) [reply packages/client/webpack.config.js:115]:
```
> { node-fetch: path.resolve('./src/shim/node-fetch.ts') }
({ node-fetch: path.resolve('./src/shim/node-fetch.ts') })
       ^

Uncaught SyntaxError: Unexpected token '-'
```

File: packages/dev-config/ts/tsconfig.node.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "https://json.schemastore.org/tsconfig",
     "display": "Node 14 – Streamr",
```
Comment (teogeb) [inline packages/dev-config/ts/tsconfig.node.json]:
```suggestion
    "display": "Node 16 – Streamr",
```

**Review by teogeb — APPROVED** [review]
👍

**Pull request merged**